plugins/s3/src/index.mjs

```diff
@@ -147,6 +273,30 @@ const DeleteObjects = {
   },
 }
 
+const GetBucketAccelerateConfiguration = {
+  awsDoc: docRoot + 'API_GetBucketAccelerateConfiguration.html',
+  validate: {
+    Bucket,
+    ...getValidateHeaders('ExpectedBucketOwner', 'RequestPayer'),
+  },
+  request: (params, utils) => {
+    const { host, pathPrefix } = getHost(params, utils)
+    return {
+      path: '/?accelerate',
+      host,
+      pathPrefix,
+      headers: { ...getHeadersFromParams(params) },
+    }
+  },
+  response: ({ headers, payload }) => {
+    return {
+      Status: payload.Status,
+      ...parseHeadersToResults({ headers }),
+    }
+  },
+  error: defaultError,
```

Comment:
Probably we can remove `defaultError` as of https://github.com/architect/aws-lite/commit/b4e16f07719fbf19a56b3078ddfb5eaa84f21981 – but please do verify!